### ISSUE TITLE
Tag Edito do not work

### ISSUE THREAD

**Issue by sonalder-darlene** [MuntashirAkon/Metro#33]
I've tried to edit tag of a mp3... it just don't work. After having clicked on the SAVE button it say scan 1 to 1 and nothing changes. I've tried to edit from the album and from the song.

**Comment by MuntashirAkon** [comment]
Was the file in question located in SD Card?

**Comment by sonalder-darlene** [comment]
> Was the file in question located in SD Card?

No, the internal storage

**Comment by MuntashirAkon** [comment]
> No, the internal storage

Weird. It's working fine on my Android 10 device.

**Comment by Liamballin** [comment]
Also doesn't work on my S7, running Android 8
Editing tags doesn't work, and when it opens all fields are empty even though the track already has some existing data.

**Comment by sonalder-darlene** [comment]
I'm running Android 10 on a custom ROM, it now works for me. The problem was the file extension or format. Previously an .opus, now a .m4a (AAC) and I've successfuly edited tags !

**Comment by Liamballin** [comment]
Correction, tag editor works for me on Android 8, but changes only show up after restarting app

**Comment by TheOPtimal** [comment]
Yes, the changes are displayed only when the app is restarted. Note that you have to clear it from recents.
